test/unit/gateways/flex_charge_test.rb

```diff
@@ -153,6 +153,15 @@ def test_address_names_from_credit_card
     assert_equal 'Doe', names.last
   end
 
+  def test_successful_store
+    response = stub_comms do
+      @gateway.store(@credit_card, @options)
```

Comment:
You can check the request construction with the check_request.

something like 
```ruby
 response = stub_comms do
      @gateway.store(@credit_card, @options)
      end.check_request do |endpoint, data, _headers|
      if endpoint == endpoint_for_access_token_response
       #some assertions
       elsif endpoint == enpoint_for_store_response
       # some assertions
       end
    end.respond_with(successful_access_token_response, successful_store_response)
```

Or even ignore the responses if this approach with two responses is creating some issues.

Reply:
The validations you propose are already carried out in other tests, from my point of view in this case they do not add extra value.

Reply:
So the main purpose of this test is the authorization value generated?